modules/go-health-monitoring.adoc

```diff
@@ -0,0 +1,19 @@
+[id="health-information-resources_{context}"]
+= Checking health information
+
+.Prerequisites
+
+* {gitops-title} Operator installed from *OperatorHub*.
+* Argo CD applications are in sync.
+
+.Procedure
+
+* The Operator will install the backend service in `openshift-gitops` namespace. Click *Environments* under the *Developer* perspective. 
+
+*  The *Environments* page shows the list of applications along with their *Environment status*.
+
+* Hover over the icons under the *Environment status* column to see the sync status of all the environments.
+
+* Click on the application to view the details of the application.
```

Comment:
seems like fluff to me, on top of that it's rather unspecific.

Reply:
Unable to understand the comment

Reply:
Sorry for not having formulated this comment very clearly.
This sentence repeats "application" twice, that seems like fluff to me. Also, it doesn't state which application, if it's the user's application it might make things more specific to state "your application". And if the user is looking at general details the sentence could go along these lines: "Click on your application to view its general data"